### DISCUSSION TITLE
Welcome to super_unko Discussions!

### DISCUSSION THREAD

**Discussion by greymd** [unkontributors/super_unko#96] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by greymd** [comment]
~うんこ~

あれ、他の人のうんこ編集できてしまう？（ @horo17  )

**Comment by jiro4989** [comment]
~~うんこ~~
↑
うんちじゃだめなんですか？(by @greymd)
↑
super_unkoなのでうんこじゃないとダメです (by @jiro4989)

**Reply by 3socha** [reply]
https://github.com/unkontributors/super_unko/blob/oshiri/lib/super_unko/unko.csv#L2-L3
どっちもあり

**Reply by jiro4989** [reply]
うんちのほうの行を消して隠蔽工作しなきゃ...
